clouddriver-core/src/main/groovy/com/netflix/spinnaker/clouddriver/model/ServerGroupManager.java

```diff
@@ -33,7 +33,7 @@
 
   Moniker getMoniker();
 
-  Set<ServerGroupSummary> getServerGroups();
+  Set<? extends ServerGroupSummary> getServerGroups();
```

Comment:
Don't worry, Deck won't trust that it's a Kubernetes SGM and [re-cast](https://github.com/spinnaker/deck/blob/master/app/scripts/modules/kubernetes/src/serverGroupManager/details/details.controller.ts#L163) it itself anyway 😂